### ISSUE TITLE
Deepslate Brick Balustrade Bridge Crash

### ISSUE THREAD

**Issue by Chip88898** [sketchmacaw/MacawsModsIssues#388]
**Describe the bug**
Upon placing the Deepslate Brick Balustrade Bridge piece, the game crashes. If you try to rejoin the game will instantly crash again, forcing the deletion of the mod to gain entry or having somebody destroy the block by using a projectile from out of render distance. 

**Mod name & Minecraft Version and Modloader**
Macaw's Bridges, 1.20.1, Forge

**To Reproduce**
1. Place down Deepslate Brick Balustrade Bridge
2. Game crashes

**Additional context**
Error Message:
The game crashed whilst rendering client block
Error: java.lang.IllegalArgumentException: Cannot get property EnumProperty{name=half, clazz=class net.minecraft.world.level.block.state.properties.Half, values=[top, bottom]} as it does not exist in Block{mcwbridges:balustrade_deepslate_bricks_bridge}

**Comment by sketchmacaw** [comment]
Can you send me the whole log? It looks like another mod is causing it

**Comment by Chip88898** [comment]
> Can you send me the whole log? It looks like another mod is causing it

I believe this is the log, now that I'm looking at it, it definitely seems to be something to do with Panda's Details trapdoor block. I'm unsure as to why that interferes with the Balustrade bridge in particular. (After further testing the same thing happened with the cobblestone balustrade, assuming its the same for all of them)
[crash-2024-07-07_19.24.16-client.txt](https://github.com/user-attachments/files/16122094/crash-2024-07-07_19.24.16-client.txt)

**Comment by sketchmacaw** [comment]
Yeah, it looks like some weird interference. The Bridge has no property in the code called "half" and some mod thinks it does have it.

**Comment by Bucert** [comment]
Same issue does aanyone have an anwser
